lib/management/configuration.js

```diff
@@ -256,12 +259,19 @@ function saveConfigurationVersion(cachedOverlay, remoteOverlay, log, cb) {
         metadata.putObjectMD(managementDatabaseName, objName, remoteOverlay,
             {}, log, error => {
                 if (error) {
-                    log.error('could not save configuration version',
+                    log.error('could not save configuration',
```

Comment:
It's not new with this change but since there's already a diff here: we should not continue to update the version pointer if there was a failure saving the overlay in the first place.